static/js/github-configuration.js

```diff
@@ -38,3 +38,27 @@ $('.logout-link').click(function (event) {
 		this.close()
 	}, 100)
 })
+
+$(".sync-connection-link").click(function (event) {
```

Comment:
If a sync fails we now display a 'Retry' option on this page. This is the click event that fires it.

Reply:
Lol, I didn't know we use jquery! Ah, good old 2004th :aww: